src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -541,14 +542,23 @@ export async function refundRequestIsValid(
     };
   }
 
-  // Validate relayer and depositId.
+  if (previousIdenticalRequests.gt(0)) {
```

Comment:
Is this true @mrice32 ? We should only handle refunds where `previousIdenticalRequests == 0`?

Reply:
I think so, at least to start. We should probably log an error (page) in this case so we can figure out why it was double requested and if the initial one was really thrown out.

The main issue this tries to handle is timing issues where the initial request was too early and was thrown out due to a lack of a match.

How do you think we should handle?

Reply:
I agree with you about logging a warning but assuming that only the first one is valid for now; that's the solution I've implemented.